docs/general/builtins/scalar-functions.rst

```diff
@@ -2196,14 +2200,19 @@ See below for an example::
 
 .. _scalar-log:
 
-``log(x : number, b : number)``
+``log(x : number[, b : number])``
 -------------------------------
 
 Returns the logarithm of given ``x`` to base ``b``.
 
-Returns: ``double precision``
+Returns: ``numeric`` or ``double precision``
 
-See below for an example, which essentially is the same as above::
+When the second argument (``b``) is provided it returns a value of type
+``double precision``. When it's not provided, then the return value will be of
```

Comment:
>When the second argument (``b``) is provided it returns a value of type ``double precision``

So basically (numeric, number) would throw an error that it's unsupported? Probably need to extend the sentence above like ... even if 'x' is of the NUMERIC type.

Reply:
thx, added clarification. If the 2nd arg is provided `x` is simply casted to double, no error.